03-JS-II/homework/tests/JSII.test.js

```diff
@@ -176,9 +176,12 @@ describe ('esVerdadero(valor)', function() {
   });
 });
 
-describe ('tablaDelSeis()', function() {
-  it('should return multiplication table of 6', function() {
-    expect(tablaDelSeis()).toEqual([0 , 6, 12, 18, 24, 30, 36, 42, 48, 54, 60]);
+describe ('factorial()', function() {
```

Comment:
Fijate que debería haber un `it` por cada tests.
Porque acá si falla cualquiera de esos te falla todo el test.

Reply:
listo, agregue los it respectivos a cada condicion